pkg/ddc/alluxio/engine.go

```diff
@@ -39,17 +35,15 @@ type AlluxioEngine struct {
 	runtimeType string
 	Log         logr.Logger
 	client.Client
-	//When reaching this gracefulShutdownLimits, the system is forced to clean up.
+	//GracefulShutdownLimits is reached, and then the system is forced to clean up.
```

Comment:
```suggestion
	//gracefulShutdownLimits is the limit for the system to forcibly clean up.
```
Maybe this is better. Actually leaving this comment as it was is also okay to me.

Reply:
Thank you. I have fixed this problem.